src/distance/edge-filters.hpp

```diff
@@ -0,0 +1,79 @@
+#ifndef SRC_DISTANCE_EDGE_FILTERS_HPP_
+#define SRC_DISTANCE_EDGE_FILTERS_HPP_
+
+#include <memory>
+#include <optional>
+
+#include "command-line/parsing/options.hpp"
+#include "common/pipeline/pipelines.hpp"
+#include "distance/edge.hpp"
+
+namespace found {
+
+
+using EdgeFilteringAlgorithms = ModifyingPipeline<Points>;
```

Comment:
I much prefer typedef here, since it was made for this (so `typedef ModifyingPipeline<Points> EdgeFilteringAlgorithms`).